### PR TITLE
add ios ui

### PR DESCRIPTION
**Pull request by yujinjcho**
https://github.com/yujinjcho/movie_recommendations/issues/24

### CODE REVIEW

File: ios_ui/MovieRec/Rating.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    //MARK: Properties
+
+    var title: String
+    var rating: String
+    var deviceId: String
+    
+    init(title: String, rating: String, deviceId: String) {
+        self.title = title
+        self.rating = rating
+        self.deviceId = deviceId
```
Comment (achiu8) [inline ios_ui/MovieRec/Rating.swift:22]: What's this for?

File: ios_ui/MovieRec/RateViewController.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        
+        // set name to be name of movie
+        titleNameLabel.text = currentMovie.title
+        
+        // set photo to be movie
+        // titleImage.image = currentMovie.photo
+        let url = URL(string: currentMovie.photoUrl)
+        // titleImage.image = UIImage(data: data!)
+        
+        DispatchQueue.global().async {
```
Comment (achiu8) [inline ios_ui/MovieRec/RateViewController.swift:65]: Is this just fetching / loading the image?

**Review by achiu8 — COMMENTED** [review]
Did you use a tutorial / project boilerplate for this?

**Comment by yujinjcho** [comment]
yeah pretty much just followed https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/

File: ios_ui/MovieRec/Rating.swift (hunk 1 above)
Comment (yujinjcho) [reply ios_ui/MovieRec/Rating.swift:22]: the deviceId? I think same way as you'd use the user_id

File: ios_ui/MovieRec/RateViewController.swift (hunk 2 above)
Comment (yujinjcho) [reply ios_ui/MovieRec/RateViewController.swift:65]: yeah. got it from the second answer. https://stackoverflow.com/questions/24231680/loading-downloading-image-from-url-on-swift

**Pull request merged**